### ISSUE TITLE
Issue with Chrome on iOS

### ISSUE THREAD

**Issue by brandonparsons** [jpillora/xdomain#76]
Please see http://stackoverflow.com/questions/23445743/stumped-re-chrome-ios-behavior

That issue could only be fixed by removing xDomain.  Is there a working fix to this?

**Comment by jpillora** [comment]
Confirmed it's not working under Chrome iOS. Working safari iOS + desktop Chrome. Very strange... Looking into a fix...

**Comment by jpillora** [comment]
So, I've got a possible temporary work around:

``` js
    setTimeout(function() {
      $("#cross").load('http://s3.amazonaws.com/jpillora-usa/xdomain/data2.json');
    }, 0);
```

On [the demo](http://jpillora.com/xdomain) page, I've simply delayed the cross-domain load until next tick and that seemed to fix it.

I'm quite sure chrome is mistakenly thinking this is a fishing attack or something

**Comment by brandonparsons** [comment]
Cool - thanks. I had removed the requirement for a cross domain call (probably a better way to go in that case) but I'll have more coming up so I'll try to remember this :)

**Comment by jpillora** [comment]
Was debugging this a bit more. If we run:

``` js
xhook.disable(); //reverts to the native XMLHttpRequest object
```

right after loading `xhook` and `xdomain`, chrome doesn't error. Will look into this again when I can

**Comment by mikesnare** [comment]
@jpillora Any progress on this?  I'm getting bit by this as well.

**Comment by sheldonbaker** [comment]
I'm stuck on this too -  any updates?

**Comment by jpillora** [comment]
Hey guys, still trying to find a work around for this >.<

Since this is actually a bug with XHook, I'll lock this thread and we'll continue the conversation here: https://github.com/jpillora/xhook/issues/23

**Comment by jpillora** [comment]
Current work-around: https://github.com/jpillora/xhook/issues/23#issuecomment-52386759

**Comment by jpillora** [comment]
Update: https://github.com/jpillora/xhook/issues/23#issuecomment-53959991

**Comment by jpillora** [comment]
Fixed in `0.6.13`
